modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java

```diff
@@ -96,6 +102,19 @@ public void testGetIdPWithInvalidId() {
         validateErrorResponse(response, HttpStatus.SC_NOT_FOUND, "IDP-60002", "random-id");
     }
 
+    @Test
+    public void testInvalidSearchAllIdPs() throws XPathExpressionException {
+
+        Response response = getResponseOfGetWithQueryParams(IDP_API_BASE_PATH, Collections.singletonMap("filter",
```

Comment:
getResponseOfGetWithQueryParams -> getResponseOfGetIDPWithQueryParams

Reply:
getResponseOfGetWithQueryParams is a method RESTTestsBase.java. Since we are using this class with all the other components I think it is better to keep the name as it is.